src/modules/core/components/PermissionRequiredInfo/PermissionRequiredInfo.css

```diff
@@ -0,0 +1,65 @@
+.title {
+  margin-bottom: 6px;
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  color: var(--dark);
+}
+
+.requiredPermissionSection {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.section {
+  display: flex;
+  flex-grow: 1;
+  align-items: center;
+  padding: 15px 0;
+  border-top: 1px solid color-mod(var(--temp-grey-blue-7) alpha(15%));
+  border-bottom: 1px solid color-mod(var(--temp-grey-blue-7) alpha(15%));
+}
+
+.sectionLabel {
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  color: var(--temp-grey-blue-7);
+}
+
+.labelIcon {
+  margin-left: 8px;
+}
+
+.labelIcon svg {
+  fill: var(--temp-grey-1);
+  stroke: var(--colony-white);
```

Comment:
Please remove the stroke on the info icon, not just set it to white.

Currently:
![Screenshot from 2020-12-08 15-34-06](https://user-images.githubusercontent.com/1193222/101490376-ebc7f980-396a-11eb-9570-2be49e48f5d4.png)

With the strike removed:
![Screenshot from 2020-12-08 15-35-17](https://user-images.githubusercontent.com/1193222/101490417-fd110600-396a-11eb-8fa9-e053fe24f348.png)